app/controllers/apo_controller.rb

```diff
@@ -87,8 +87,16 @@ def set_apo_metadata apo, md_info
     apo.metadata_source      = md_info[:metadata_source]
     apo.agreement            = md_info[:agreement].to_s
     apo.default_workflow     = md_info[:workflow ] unless (!md_info[:workflow] || md_info[:workflow].length < 5)
-    apo.use_license          = md_info[:use_license]
     apo.default_rights       = md_info[:default_object_rights]
+    if md_info[:use_license].blank?
+      # delete use licenses, ensure defaultObjectRights datastream is saved correctly
+      apo.defaultObjectRights.creative_commons = nil
+      apo.defaultObjectRights.open_data_commons = nil
+      apo.defaultObjectRights.creative_commons_human = nil
```

Comment:
Ideally, the "human" assignments should already be accomplished by the earlier setter methods, right?  This would follow the pattern set in Editable.  The reason being, it shouldn't be possible to set the code and still have an out-of-date or mismatched human readable value.  (Personally, I'd prefer not storing the "human" readable (i.e. *English*) version at all, but can punt on that.)